packages/comparisons/src/data.json

```
@@ -18102,6 +18233,19 @@
 			}
 		]
 	},
+	{
+		"eslint": [
+			{
+				"name": "@typescript-eslint/no-unsafe-type-assertion",
+				"url": "https://typescript-eslint.io/rules/no-unsafe-type-assertion"
+			}
+		],
+		"flint": {
+			"name": "unsafeTypeAssertions",
```

I've been trying to put the subject of the rule (e.g. type assertions) first, before the descriptor (e.g. safety). `valueSafety` is another one.

```suggestion
			"name": "typeAssertionSafety",
```

---

I think so, yes. Either in this PR or a follow-up issue, up to you.